src/Microsoft.SqlTools.ServiceLayer/QueryExecution/Contracts/QueryExecuteRequest.cs

```diff
@@ -32,6 +32,12 @@ public class QueryExecuteResult
         /// Connection error messages. Optional, can be set to null to indicate no errors
         /// </summary>
         public string Messages { get; set; }
```

Comment:
I'll quickly switch the parameter out to have a message type instead - I like that idea (and it's an easy change)